Simplenote/Classes/SPBlurEffectView.swift

```diff
@@ -0,0 +1,111 @@
+import Foundation
+import UIKit
+
+
+// MARK: - SPBlurEffectView: Reacts automatically to UserInterfaceStyle Changes
+//
+@objc
+class SPBlurEffectView: UIVisualEffectView {
+
+    /// Blur's TintView
+    ///
+    private let tintView: UIView = {
+        let output = UIView()
+        output.autoresizingMask = [.flexibleWidth, .flexibleHeight];
+        return output
+    }()
+
+    /// Closure that's expected to return the TintColor we want. Runs on `iOS <13` everytime the active Theme is updated.
+    ///
+    var tintColorClosure: (() -> UIColor)? {
+        didSet {
+            refreshTintColor()
+        }
+    }
```

Comment:
`Closure` is already encapsulated in the type here - the name is redundant.

<img width="583" alt="Screen Shot 2019-11-08 at 3 39 39 PM" src="https://user-images.githubusercontent.com/5431237/68512439-04348480-023e-11ea-884b-41d906bade17.png">

Is it our practice to do this or would it be normal to also choose to name this `tintColor` or `getTintColor`?

Reply:
Thing is: the property contains a `closure` that gives you a `tintColor`. It's not a `tintColor` directly, so I wanted to be extra clear with the name!

Reply:
<img width="711" alt="Screen Shot 2019-11-11 at 10 58 46 AM" src="https://user-images.githubusercontent.com/5431237/68605571-40e8c180-0472-11ea-837b-da6fec4e85f4.png">

just curious

Reply:
in that case, you'd need to do this:

```
tintView.backgroundColor = tintColorClosure?()
```

(Since it's a pointer to an anonymous function "closure"), assigning it plainly to a property that expects a UIColor is expected to break.

Reply:
That assignment reads as:

```
1. If there's a closure set in this property (that's the ? part)
2. Then execute (that's the () part)
3. Otherwise, nil is assigned
```

Reply:
right, I just meant that the IDE already makes it obvious if I use a closure under the assumption that it's a property. it won't compile, but before I even try it tells m "you doofus, you can't do that"